### ISSUE TITLE
Error spam in console

### ISSUE THREAD

**Issue by freethemice** [xenondevs/Nova#79]
### Agreements

- [X] I am running the latest version of Nova.
- [X] I am running the latest version of my server software.
- [X] I have checked for similar issues.

### Other plugins

This bug also occurs when no other plugins are present.

### Plugin Version, Server Software, Server Version

nova 0.7.12, paper 1.81.1

### Observed/Actual behavior

logged in this morning and the console started spamming this.

### Steps to reproduce

1. I just starting playing with the mod yesterday, don't have that much setup. block breaker, furnace gen, press machine, furnaces, elite cables
2. every thing was working fine yesterday, and is far as I can tell everything is still working. just the error in console

### Errors

https://pastebin.com/xyvNFNc0

**Comment by ByteZ1337** [comment]
Could you send a screenshot of the setup at x: 8, y: 79, z: -4998.0?

**Comment by freethemice** [comment]
I broke it after posting the error, I had everything going through one elite cable, items and power. on 4 different channels. 
1 had 5 furnace generators, 1 block breaker, 5 electrical furnaces, 2 Mechanical press, everything was going in and out of a 1 double chest that had 3 cables connected. this chest had everything in it , coal, raw ore, and ingots. 
also, the error stopped when I broke everything.

I don't know if this is related but the item filters didn't drop when I broke the old basic cables, and after I placed elite cables the old Item filters where still in them.
